src/lv_message_efficient.cc

```diff
@@ -57,7 +57,6 @@ namespace grpc_labview
     DEFINE_PARSE_FUNCTION(uint64_t, UInt64, UINT64, UInt64)
     DEFINE_PARSE_FUNCTION(float, Float, FLOAT, Float)
     DEFINE_PARSE_FUNCTION(double, Double, DOUBLE, Double)
-    DEFINE_PARSE_FUNCTION_SPECIAL(int32_t, Enum, ENUM, Enum)
```

Comment:
Note: There is now a custom implementation of this rather than using this macro so we can perform translation between proto enum value and LV enum values.